Gopkg.toml

```diff
@@ -5,192 +5,84 @@ required = [
-
-[[override]]
-name="github.com/json-iterator/go"
-version="1.1.3"
-
-[[override]]
-name="github.com/mailru/easyjson"
-revision="8b799c424f57fa123fc63a99d6383bc6e4c02578"
-
-[[override]]
-name="github.com/modern-go/concurrent"
-version="1.0.3"
-
-[[override]]
-name="github.com/modern-go/reflect2"
-version="1.0.0"
-
-[[override]]
-name="github.com/onsi/ginkgo"
-version="v1.4.0"
-
-[[override]]
-name="github.com/onsi/gomega"
-version="v1.3.0"
+  name="sigs.k8s.io/controller-runtime"
+  version="v0.1.1"
 
-[[override]]
-name="github.com/spf13/pflag"
-version="v1.0.1"
-
-[[override]]
-name="golang.org/x/crypto"
-revision="4ec37c66abab2c7e02ae775328b2ff001c3f025a"
+[[constraint]]
+  name="k8s.io/api"
+  version="kubernetes-1.10.1"
 
-[[override]]
-name="golang.org/x/net"
-revision="640f4622ab692b87c2f3a94265e6f579fe38263d"
+[[constraint]]
+  name="k8s.io/apiextensions-apiserver"
+  version="kubernetes-1.10.1"
 
-[[override]]
-name="golang.org/x/oauth2"
-revision="cdc340f7c179dbbfa4afd43b7614e8fcadde4269"
+[[constraint]]
+  name="k8s.io/apimachinery"
+  version="kubernetes-1.10.1"
 
-[[override]]
-name="golang.org/x/sys"
-revision="7db1c3b1a98089d0071c84f646ff5c96aad43682"
+[[constraint]]
+  name="k8s.io/code-generator"
+  version="kubernetes-1.10.1"
 
-[[override]]
-name="golang.org/x/text"
-version="v0.3.0"
+[[constraint]]
+  name="k8s.io/client-go"
+  version="kubernetes-1.10.1"
 
-[[override]]
-name="golang.org/x/time"
-revision="fbb02b2291d28baffd63558aa44b4b56f178d650"
+[[constraint]]
+  name = "github.com/onsi/ginkgo"
+  version = "v1.5.0"
 
-[[override]]
-name="google.golang.org/appengine"
-version="v1.0.0"
+[[constraint]]
+  name = "github.com/onsi/gomega"
+  version = "v1.4.0"
 
-[[override]]
-name="gopkg.in/inf.v0"
-version="v0.9.1"
+[[constraint]]
+  name = "github.com/spf13/afero"
+  version = "v1.1.1"
 
-[[override]]
-name="gopkg.in/yaml.v2"
-version="v2.2.1"
+[[constraint]]
+  name = "gopkg.in/yaml.v2"
+  version = "v2.2.1"
 
-[[override]]
-name="k8s.io/api"
-version="kubernetes-1.10.0"
+[[constraint]]
+  name = "github.com/emicklei/go-restful"
+  version = "v2.7.1"
 
-[[override]]
-name="k8s.io/apiextensions-apiserver"
-version="kubernetes-1.10.1"
+[[constraint]]
+  name = "github.com/spf13/cobra"
+  version = "v0.0.3"
 
-[[override]]
-name="k8s.io/apimachinery"
-version="kubernetes-1.10.0"
+[[constraint]]
+  name = "github.com/spf13/pflag"
+  version = "v1.0.1"
 
-[[override]]
-name="k8s.io/client-go"
-version="kubernetes-1.10.1"
+[[constraint]]
+  name = "github.com/spf13/viper"
+  version = "v1.0.2"
 
-[[override]]
-name="k8s.io/kube-aggregator"
-version="kubernetes-1.10.1"
+[[constraint]]
+  name = "github.com/ghodss/yaml"
+  version = "1.0.0"
 
-[[override]]
-name="k8s.io/kube-openapi"
-revision="f08db293d3ef80052d6513ece19792642a289fea"
+# TODO(droot): pin the following deps to a version when they are tagged.
+#[[constraint]]
+#  name = "github.com/go-openapi/spec"
+#
+#[[constraint]]
+#  name = "github.com/golang/glog"
```

Comment:
To be consistent with `controller-runtime`, `controller-tools` should migrate to `logr` from `glog`.
cc: @DirectXMan12

Reply:
Removed it. Using standard 'log' in this case because use-case is print something on stdout as part of CLI command.